cosh.json

```diff
@@ -18,12 +18,13 @@
         }
     ],
     "returns": {
-        "description": "The computed hyperbolic cosine of `x`.",
+        "description": "The computed hyperbolic cosine in the range *[1, +∞)*.",
```

Comment:
(can't comment on lines above)

as mentioned earlier, I would drop "works on radians only" and describe the "x" parameter as "Hyperbolic angle"

Reply:
Done, I think...